src/CommonAssemblyInfo.cs

```diff
@@ -3,9 +3,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
-[assembly: AssemblyInformationalVersion("2.1.0")]
+[assembly: AssemblyVersion("2.2.0")]
+[assembly: AssemblyFileVersion("2.2.0")]
+[assembly: AssemblyInformationalVersion("2.2.0")]

```

Comment:
The technique is sound, but if we're only ever updating the version via 
`rake next_version[new_version]`, I don't think we'll run into problems.
And I think we'd lose the ability to leverage albacore for this (which we may eventually anyhow, if/when we drop rake).
Still, if you're keen on it, I see no other downside.
I'd prefer to make it its own issue, though, rather than doing as part of the last step of closing off 2.1.0.